xml/libvirt_networking.xml

```diff
@@ -8,958 +8,976 @@
-      Address.
+      The following configuration routes traffic from the virtual network to
+      the LAN without applying any NAT. The IP address range must be
+      preconfigured in the routing tables of the router on the &vmhost;
+      network.
      </para>
-    </step>
-    <step>
+<screen>
+&lt;network>
+ &lt;name>vnet_routed&lt;/name>
+ &lt;bridge name="virbr1" />
+ &lt;forward mode="route" dev="eth1"/><co xml:id="vnet-xml-route"/>
+ &lt;ip address="192.168.122.1" netmask="255.255.255.0">
+  &lt;dhcp>
+   &lt;range start="192.168.122.2" end="192.168.122.254" />
+  &lt;/dhcp>
+ &lt;/ip>
+&lt;/network>
+</screen>
+     <calloutlist>
+      <callout arearefs="vnet-xml-route">
+       <para>
+        The guest traffic may only go out via the <systemitem>eth1</systemitem>
+        network device on the &vmhost;.
+       </para>
+      </callout>
+     </calloutlist>
+    </example>
+    <example>
+     <title>Isolated Network</title>
      <para>
-      Select the desired method for assigning an IP address to the VLAN device.
+      This configuration provides a completely isolated private network. The
+      guests can talk to each other, and to &vmhost;, but cannot reach any
+      other machines on the LAN, as the &lt;forward> element is missing in the
+      XML description.
      </para>
-    </step>
-    <step>
+<screen>&lt;network>
+ &lt;name>vnet_isolated&lt;/name>
+ &lt;bridge name="virbr3" />
+ &lt;ip address="192.168.152.1" netmask="255.255.255.0">
+  &lt;dhcp>
+   &lt;range start="192.168.152.2" end="192.168.152.254" />
+  &lt;/dhcp>
+ &lt;/ip>
+ &lt;/network>
+</screen>
+    </example>
+    <example>
+     <title>Using an Existing Bridge on &vmhost;</title>
      <para>
-      Click <guimenu>Next</guimenu> to finish the configuration.
+      This configuration shows how to use an existing &vmhost;'s network bridge
+      <literal>br0</literal>. &vmguest;s are directly connected to the physical
+      network. Their IP addresses will all be on the subnet of the physical
+      network, and there will be no restrictions on incoming or outgoing
+      connections.
      </para>
-    </step>
-   </procedure>
-   <para>
-    It is also possible to use the VLAN interface as a physical interface of a
-    bridge. This makes it possible to connect several &vmhost;-only networks
-    and allows to live-migrate &vmguest; systems that are connected to such a
-    network.
-   </para>
-   <para>
-    &yast; does not always allow to set no IP address. However, this may be a
-    desired feature especially if &vmhost;-only networks should be connected.
-    In this case, use the special address <literal>0.0.0.0</literal> with
-    netmask <literal>255.255.255.255</literal>. The system scripts handle this
-    address as no IP address set.
-   </para>
+<screen>&lt;network>
+        &lt;name>host-bridge&lt;/name>
+        &lt;forward mode="bridge"/>
+        &lt;bridge name="br0"/>
+&lt;/network>
```

Comment:
```suggestion
<screen>&lt;network&gt;
        &lt;name&gt;host-bridge&lt;/name&gt;
        &lt;forward mode="bridge"/&gt;
        &lt;bridge name="br0"/&gt;
&lt;/network&gt;
```

Reply:
There are several instances of '>' vs '&gt;' in a similar context, e.g. lines 745-760, 830-839 and 858-866. Should those be changed as well?

Reply:
Yes, I think they should.

Reply:
Thanks. I've changed all occurrences and force pushed the latest patch. I think I've covered all review comments.